src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/carts.yml

```diff
@@ -68,20 +68,44 @@ admin_carts_set_free_shipping:
   options:
     expose: true
 
-admin_carts_add_rule:
-  path: /{cartId}/rules/add
+admin_carts_add_cart_rule:
+  path: /{cartId}/cart-rules
```

Comment:
I think you should keep the `add` prefix, so it is clear *one* cart rule is added, and not we update *all* the cart rules The `cart-rules` modification is better though! so it would give `/{cartId}/cart-rules/add` which matches the format of the delete urls

Reply:
idk, this way seemed to suit api standards better 😄 I was suggested by @sarjon, so wdyt ? 😄

Reply:
The thing is I wonder if we have a defined standard on these API calls ^^ This should be decided and documented I guess so that all our APIs follow the same convention

Reply:
> I wonder if we have a defined standard on these API calls

Not that I aware of.

> This should be decided and documented I guess so that all our APIs follow the same convention

You are right, but for now I think we are trying to follow (more or less) standard Rest API naming, see https://restfulapi.net/resource-naming/